common/src/main/java/com/example/demo/ShopApiEndpoints.java

```diff
@@ -0,0 +1,28 @@
+package com.example.demo;
+
+
+/**
+ * <p>
+ * <p>
+ * Centralise l'ensemble des endpoints disponibles pour les apis RESTS.
+ * Utilisée dans
+ * <ul>
+ * <li>la définition des apis</li>
+ * <li>les tests</li>
+ * </ul>
+ * </p>
+ */
+public class ShopApiEndpoints {
+
+    private ShopApiEndpoints() {
+        throw new AssertionError("Utility class must not be instantiated.");
+    }
+
+    public static final String ROOT = "/";
+
+    public static final String SHOES = "/shoes";
+    public static final String SEARCH = "/search";
+    public static final String SHOES_SEARCH = SHOES.concat(SEARCH);
+
+    public static final String STOCK = "/stock";
+}
```

Comment:
My idea was to centralize all messages in internalization files and use I18n Spring.
The first thing is the simplicity of maintaining the messages (but the deadline).
Second, because a large part of the messages will be exposed to the consumer of our API (client) and in the logs, this makes it easier to understand error messages.

I agree with you, error messages should be centralized in Enums.
Github issue is created #17 .